website/routes.py

```diff
@@ -65,6 +66,7 @@ def get_globals():
         'user_timezone': user.timezone if user and user.timezone else '',
         'user_url': user.url if user else '',
         'user_gravatar': profile_views.current_user_gravatar(size=25)['gravatar_url'] if user else '',
+        'user_email_verifications': auth_views.verified_email_get(uid=user._id) if user else [],
```

Comment:
You shouldn't need to call a view function; just use a normal function or a property of `User`. As suggested in a previous comment, the `verified_email_get` endpoint doesn't appear to be used anywhere. Just turn it into a property, and you'll be golden.